internal/venice-common/src/main/java/com/linkedin/venice/pushmonitor/ReplicaStatus.java

```diff
@@ -47,14 +47,16 @@ public ReplicaStatus(String instanceId, boolean enableStatusHistory) {
     this.statusHistory = enableStatusHistory ? new LinkedList<>() : null;
   }
 
-  public void updateStatus(ExecutionStatus newStatus) {
-    currentStatus = newStatus;
+  public void updateStatus(ExecutionStatus newStatus, String incrementalPushVersion) {
+    setIncrementalPushVersion(incrementalPushVersion);
```

Comment:
Do we need the replica-level incremental push version field?
I think this should be only required in the snapshot history, right?

Reply:
Good question, let me check the usage

Reply:
No, I think this is misleading...Venice has been using this field to record different kinds of errors and different kinds of messages. Personally I am against this mixed usage, but I think the cleanup (or renaming) can happen in a separate PR as it is all over the places..
Anyway, for inc push, we are not using this field to determine job status